torch/_inductor/comm_analysis.py

```diff
@@ -49,30 +49,30 @@
 from .virtualized import V
 
 
-class NCCL_COLL(Enum):
+class NCCL_COLL(IntEnum):
     ALL_REDUCE = 0
     ALL_GATHER = 1
     REDUCE_SCATTER = 2
 
 
-class NCCL_HW(Enum):
+class NCCL_HW(IntEnum):
     NVLINK = 0
     PCI = 1
     NET = 2
 
 
-class NCCL_ALGO(Enum):
+class NCCL_ALGO(IntEnum):
     TREE = 0
     RING = 1
 
 
-class NCCL_PROTO(Enum):
-    SIMPLE = 0
-    LL = 1
-    LL128 = 2
+class NCCL_PROTO(IntEnum):
+    LL = 0
```

Comment:
nit: could you add some comment on this enum? the other enums are easy to understand, but this one requires a bit knowledge to understand